NickvisionTubeConverter.Shared/Helpers/DependencyManager.cs

```diff
@@ -68,19 +42,37 @@ public static string Ffmpeg
     }
 
     /// <summary>
-    /// Downloads dependencies (For Windows ONLY)
+    /// Setups dependencies for the application
     /// </summary>
     /// <returns>True if successful, else false.</returns>
-    public static async Task<bool> DownloadDependenciesAsync()
+    public static async Task<bool> SetupDependenciesAsync()
     {
-        if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-        {
-            return true;
-        }
         try
         {
-            await YoutubeDL.DownloadYtDlpBinary(YtdlpPath.Remove(YtdlpPath.IndexOf("yt-dlp.exe")));
-            await YoutubeDL.DownloadFFmpegBinary(Ffmpeg.Remove(Ffmpeg.IndexOf("ffmpeg.exe")));
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                await YoutubeDL.DownloadFFmpegBinary(Ffmpeg.Remove(Ffmpeg.IndexOf("ffmpeg.exe")));
```

Comment:
Probably shouldn't depend on YoutubeDLSharp solely to install FFmpeg :sweat_smile:

Reply:
Yes I will fix that before merge :)